src/main/java/fruitshop/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,20 @@
+package fruitshop.service.impl;
+
+import fruitshop.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String fileReport, String fileName) {
+        File file = new File(fileName);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new java.io.FileWriter(
+                file))) {
+            bufferedWriter.write(fileReport);
+        } catch (IOException e) {
+            throw new RuntimeException("can't write to file " + fileName, e);
```

Comment:
According to the checklist, exceptions in try-catch blocks should not be ignored or simply printed. However, it looks like you've implemented this correctly by rethrowing a RuntimeException with an informative message including the fileName, which aligns with the good practice example in the checklist.